# PhD project meeting notes 2020-10-15

Attendees: Tom B, Gib, Nic, Tom G

## Agenda

* [h<sup>2</sup><sub>EWAS</sub> name change](#name-change)
* [random thesis bits](#random-thesis-bits)
* [comments](#comments)
* [plan going forward](#plan-going-forward)
* [AOB](#aob)

## h<sup>2</sup><sub>EWAS</sub> name change <a name="name-change"></a>

* Don't mind how we name h<sup>2</sup><sub>EWAS</sub> 
	+ originally called it m<sup>2</sup>
	+ Doug asked for it to be changed to h<sup>2</sup><sub>EWAS</sub>
	+ I like the idea it has "EWAS" in the name
	+ If h<sup>2</sup> makes it seem as though we're measuring heritability of DNAm then happy to change it
	+ Nic suggested m<sub>r<sup>2</sup></sub> -- ??

__NOTES:__

Some meeting notes

## random thesis bits

* Mocks setup

* setting up one drive

* Anything else to think about at this stage?

__NOTES:__

Some meeting notes

## comments

* anything you guys want to bring up?

### overarching thoughts/important points

* Adding in a "Data sources" chapter
	+ Think that it may not be "needed" as I can put the data in the first results chapter it's used in and then just refer back to it
	+ BUT could be useful as the use of data differs between some chapters -- may be confusing saying in the EWAS Catalog chapter "we used ALSPAC data for xyz" and then having to refer back to that
	+ Also could be used to appraise each dataset?

* Updating results with new data
	+ Definitely will be updating The EWAS Catalog, BUT may not have time to update properties of ewas and ewas-gwas comp...
	+ Important to think about how to frame this in thesis

* Incorporating other peoples work into thesis:
	+ EWAS Catalog -- Team effort (James S made original website)
	+ Properties of EWAS -- Charlie has done enrichment analyses
	+ DNAm-lung cancer MR -- Becky joint first author AND Andrew Teschendorff did analysis on TCGA data

### ewas-gwas comparison

* TRG: "GO is not pathway" - very good point... GO terms place genes into "functional groups" 

* Being more cautious with end of abstract: "However, they did not preclude the posibility that some of the differentially methylated positions may be pertinent to the aetiology of the traits. Thus, combining GWAS and EWAS (amongst other studies), rather than using one study design, will likely be more fruitful in understanding trait biology." -- Agree caution is needed! Find it a little hard to strike a balance of being cautious but not implying EWAS is pointless in this thesis...

* TRG great point: Differences in properties of DNAm and genetic variants extends beyond just confounding and reverse causation -- correlation structure and ??

* Reasons to expect differences between gene and pathway overlap between EWAS and GWAS:
	+ confounding
	+ reverse causation
	+ trans mQTL effects (only differences in genes?)
	+ DNAm mediating non-genetic causes of trait (only differences in genes?) -- also question of if you had an infinite sample size would you detect all genes relevant to trait aetiology via GWAS?
	+ mis-mapping of CpGs or SNPs to genes (yes, but for intro good to assume we are not doing this? Even though of course we will be...)

* NJT great Q: "what would happen if you had infinite power for both the GWAS and the EWAS? One assumes that the overlap would then be a function of the genetic architecture of the trait of interest. If you had something truly poly/omnigenic, then you would get overlap I guess as you find “genetics of” confounding. For mono/oligogenic traits, overlap will always be capped. This may make you reflect more on the power/capacity of the EWAS and GWAS and how this may have shaped your learning here (e.g. educational attainment which you have)?"
	+ Definitely think it's worth discussing this, but in the intro or discussion?

* More ambitious with conclusion -- best way to talk about expansion to more than 7 traits?

### EWAS Catalog

* Justifying decisions made before I came on-board? -- e.g. why include only associations at P<1e-4?

* Comparison of EWAS Catalog to other datasets to get across why pooling data is useful

### Properties of EWAS

* will discuss these at next meeting! 

### Intro 

* Structure of intro to whole thesis
	+ GH: could re-structure intro so it is more high-level
	+ NJT: "opening lines are key to getting the reader with you - use turns of phrase that ensure they get that you are deeply aware of the properties of the data you are focused on for your work"
	+ agree with both -- struggled a lot to write this...

* Couple of things I didn't quite understand from comments, but can leave to next meeting
	+ including this from NJT: '“omewide” era where hypothese were abandoned for acephaly"' 

__NOTES:__

Some meeting notes

## Plan going forward

* Comments on everything apart from 1. properties of ewas work and 2. discussion

* Immediatly after meeting - will send round discussion with Gibs comments

* By end of weekend -- will incorporate some of Nic's comments on properties of EWAS (except more difficult ones I want to discuss) + send round a copy of this WITH some extra analyses I left out of first draft
	+ Think it would be nice to focus on this chapter in next meeting (Oct 27th)

* Will incorporate ewas-gwas comparison comments and put into thesis -- will then have all chapters in one place


## AOB

* AOB item1