# Introduction

This thesis focuses on epigenome-wide association studies (EWAS), which assess the association between DNA methylation changes throughout the genome and traits of interest. Advances in technology and an increasing number of samples has lead to hundreds of EWAS having been performed to date (REFERENCE EWAS CATALOG CHAPTER). Goals of these EWAS included using DNA methtlation marks as predictors, diagnostic factors, and modifiable mediators of traits. 

To help inform design of future studies, there is a need to understand what has been discovered thus far in EWAS and what future EWAS are likely to add in the context of current EWAS and other study designs. Further, to improve biological inference of sites identified in EWAS, causality (or lack of) should be established. IS THIS TOO VAGUE?

NON-VAGUE THING TO SAY -- read over both and decide what to add:
- Key things which are not currently understood:
- Why certain DNA methylation sites/regions of the genome are more prevalent in EWAS to date
- How much trait variation DNA methylation tends to associate with 
- Whether EWAS can add to current biological understanding in addition to other study designs
- Whether DNA methylation-trait associations in EWAS are causal

In this chapter I describe DNA methylation in context of the regulatory processes in human cells, discuss it's potential for use in population level research and describe the current state of EWAS research. Then I discuss how epigenetic-epidemiologists can draw on the methods developed by their genetics counter-parts to 1. understand what information has been gained from EWAS, 2. understand what information is left to gain from EWAS and 3. understand the causal nature of DNA methylation-trait associations identified in EWAS.


## DNA methylation as part of the regulome
The research questions of this thesis pertain to a specific study type, EWAS, of a specific epigenetic mark, DNA methylation. DNA methylation is one of many epigenetic marks that are involved in gene regulation (the regulome), so here I briefly outline some of these regulatory marks and discuss DNA methylation in the context of this complex biological process.

### The regulome
Gene expression is a tightly controlled process and only in the right circumstances will RNA polymerase II be able to bind the correct site, initiate and finally complete the transcription process (REFS). The importance of this level of control is no more apparent than in the developmental stages of human life. Humans start as a single cell and after roughly nine months are transformed into a multicellular organism with trillions of cells, including 100s of unique cell types (REF). As these cells arise from a single progenitor they must contain identical genetic sequences (bar a few somatic mutations). Therefore, the process by which the body is able to create such diversely functioning cells and tissues must come from regulation of how the genetic sequence is read and its products (REFs). Indeed it was in developmental biology that we first began to understand the processes that may regulate gene expression (REFs). There are a plethora of methods cells use to regulate gene and protein expression post-transcriptionally (REFs), but these are beyond the scope of this thesis. So I'll continue to describe the regulome just in the context of epigenetic marks.

### Defining epigenetics
The definition of epigenetics is much debated amongst those who study cellular and molecular processes (REFs). Waddington first coined the term and he originally defined it as __X__ (REF). To avoid confusion, throughout the rest of the thesis, here is what I mean when using the term __epigenetics__: the study of mitotically heritable changes in gene expression that occur without changes in DNA sequence. When referring to __epigenetic marks__ I simply mean chemical changes to the genome and genome-bound proteins that are mitotically heritable and may influence gene expression without changing the DNA sequence. 

### Histone modifications 
Histones are genome-bound proteins composed of four subgroups that are present twice each in a single histone octamer. Modifications can occur to any of the histone monomers and these have been associated with both positive and negative changes in gene expression (REFs). Histone modifications are numerous and complex in nature. To briefly describe the complexity, there are at least __X__ types of histone modifications, each of the histone monomers can be modified across many different sites, and for any one site mutiple of the same modification can occur and it is the combination of all of this that plays a role in gene expression regulation (REFs). Furthermore, histone modifications are subject to rapid change upon environmental stimulus to help induce or repress gene expression (REFs). This has meant, that for only a few histone marks are we certain of the role (REFs) and that taking a snapshot of the histone modifications present in cells may give a poor summary of how gene regulation is occuring. Very few population-based studies have assessed histone modifications because of these, plus some other technical difficulties (REFs). They may become far more prominent in the future as our understanding and ability to measure the modifications in a meaningful way increases. 

### DNA methylation
Good review: Peter Jones 2012
DNA methylation is the addition of a methyl group to the DNA, which primarily occurs at the 5' cytosine of a CpG site. Little is known about the role of non-CpG site DNA methylation and current EWAS only measure CpG methylation, so that will be my focus here. The function of this epigenetic mark was proposed back in 1975 (REFs), where two papers suggested it represses gene expression and ever since then many papers have shown the correlation between DNA methylations around genes and a reduction in expression of those genes. Unfortunately, it's not that simple and even now the role of DNA methylation is being debated, with a paper recently suggesting that any DNA methylation changes associated with changes in gene expression were just a by-product of other regulatory processes such as transcription factor binding (REF). Regardless, DNA methylation is closely linked with genomic function and correlates with various aspects in different ways (__FIGURE__). 

CpG sites are not randomly distributed throughout the genome, but are often found in clusters and so if DNA methylation does have an effect on things like gene expression it's thought that methylation and de-methylation of CpG sites in groups is what drives their regulatory function. To support this, there are clear biological processes that regulate DNA methylation at nearby sites together, for example, CpG sites at transcription factor binding sites will be de-methylated as a group when the transcription factor binds (REF), and further nearby sites are often statistically correlated. However, there is no evidence to suggest that neighbouring sites do indeed act in tandem or whether it is likely one site from the group driving regulatory function. This is something I explore a little more in (REFERENCE h2ewas CHAPTER!).  

One major difference between DNA methylation and other epigenetic marks, that was alluded to earlier, is that DNA methylation is far more stable. Enzymes do exist that can actively demethylate the DNA, for example the ten-eleven translocation (TET) enzymes, but ultimately cell division is required for full demethylation of a DNA molecule. Biologically, this suggests DNA methylation might be involved in long-term repression of gene expression, which can be seen for X-inactivation, and practically it makes studying the epigenetic mark easier as it prevents large temporal variations in DNA methylation (though they may occur!), so fewer samples are needed.

Figure here: 

Figure should contain these diagrams:
1. DNA methylation at TSS and no DNA methylation in gene body (repressed transcription)
2. No DNA methylation at TSS and DNA methylation in gene body (active transcription)
3. DNA methylation in centromeres
4. DNA methylation at transposable elements


## Use of DNA methylation in a population settingÂ {#dnam-phs}

The relationship between a large number of traits and DNA methylation have now been studied, from childhood adversities (REFs) to smoking (REFs), and one of the appealing factors of doing this research is that DNA methylation is modifiable. Thus, in theory, negative effects of something like childhood adversity may be captured in DNA methylation and could potentially be, in part, reversed by altering the methylome in some way. This view that clinicians may soon have the power to do anything like this has likely lead to a lot of poor and inconsequential EWAS being published. Unfortunately, there are massive complications to this research. Firstly, it should be recognised that DNA methylation has the same undesirable properties of every other complex phenotype, it correlates with a lot of things, making any associations suceptible to confounding and reverse causation, so determining causality is tricky (this will be discussed in \@ref(genetics-in-ewas)). Secondly, current methods used to modify DNA methylation mostly rely on broad remodelling of the DNA methylome (e.g. 5-AZA), which is unviable in the clinical setting for treatment of most things. Granted, there are methods that can be used to increase specificity of targetting DNA methylation changes in the genome (REFs), these have not yet been adapted for use in the clinic and given the brevity of their existence, it's unlikely this will occur soon. Finally, DNA methylation varies between tissue and cell types. Not only does this pose the issue of confounding when pooling cell types from a tissue, it also leads to large complexities in the interpretation of DNA methylation studies. Most studies are conducted in readily available tissues, such as blood and saliva, for obvious reasons, but there is no way of knowing which tissue is best suited for studying DNA methylation for many complex traits. DNA methylation does correlate between tissues (REFs), which may mean a readily available tissue can act as a surrogate, but this also means targeting DNA methylation in the tissue measured for the EWAS, may have no impact on your phenotype. All together these problems mean that establishing causality is difficult, then targeting specific DNA methylation sites of interest to effect change is also challenging (and untested in humans), and finally the tissue that needs to be targetted may be impossible to target anyway. Establishing causality in EWAS is a focus of __\@ref(dnam-lung-cancer-mr)__.

These difficulties do not exclude DNA methylation as a trait that is important to study in a population setting; it is not necessary to establish causality to use DNA methylation as a diagnostic biomarker or predictor of future disease. Further, important biological information can be gained from studying DNA methylation with relation to many traits of interest, but one must be careful how they approach and interpret their studies.

## Epigenome-wide association studies {#ewas}

Epigenome-wide associaiton studies are the flagship study design for estimating the association of DNA methylation with a trait of interest in a population setting. The premise is simple, DNA methylation is measured across parts of the whole genome, often using an array such as the Illimnum HumanMethylation 450 Beadchip (HM450), then simple regressions are conducted to estimate the association of DNA methylation at each site with a trait of interest. 


ALSO ADD IN THAT DMR ANALYSIS IS COMMON IN EWAS --> WANT TO REFERENCE H2EWAS PAPER AGAIN!

* Describe what an EWAS is and explain how it is conductedÂ 
* Describe the current successes that have come from EWASÂ 
* Describe the current problems with EWAS

## Using methods from genetics to help inform future EWAS {#genetics-in-ewas}

There is a clear correlary of the EWAS in genetics, the genome-wide association study (GWAS), which also measures markers across the whole genome and assesses whether each of these markers associates with the phenotype of interest. GWAS have been around for far longer than EWAS and a huge amount of effort has been put into understanding what information is provided by these studies, what information can be discovered by these studies and how to use these results to inform other research.

### The GWAS Catalog {#gwas-catalog}

The GWAS Catalog is a manually curated database of publically available GWAS data, developed by the EBI and made openly available to the public (REFs). It has a broad range of applications for researchers, from replication of GWAS, to identifying overlapping GWAS signals between traits, to pooling the data to try and understand the genetic architecture of complex traits as a whole. Resources like this are invaluable to the genetic epidemiologist community and so developing a corralary database for EWAS may provide equal opportunity for epigenetic epidemiologists. Catalogs such as EWASdb (REF) and the EWAS Atlas (REF) are currently available, but fall short of some researcher requirements including ease of use and access to full summary statistics. The development of a new database is the focus of __Chapter \@ref(ewas-catalog)__. 

### Total variance captured by all sites measured genome-wide {#heritability}

If at all possible, it's important to have an understanding of whether your exposure of interest covaries with the outcome of interest. Of course, if they are independent then studying the relationship between the two would be pointless. It is possible to estimate the proportion of phenotypic variance that is attributable to genetic factors by estimating heriability ($H^2$). The vast proportion of $H^2$ for any given complex trait is made up of the the additive effects $h^2$, or narrow-sense heritability. Without measuring any genotypes it is possible to estimate $h^2$ for a trait given the relatedness of individuals in a sample and an ability to minimise environmental influences, twin studies are an example of this. Simply, if monozygotic twins, who share 100% of their DNA sequence, tend to be more similar for the trait of interest than dizygotic twins, who share 50%, then the trait would be estimated to have some additive genetic component ($h^2$ > 0). The degree of difference between the estimates would provide an estimate of $h^2$ and so the contribution of all additive genetic effects to the variance of the trait. Heritability analyses have been conducted for a huge amount of traits, which provide adequate evidence that when searching for genetic effects on those traits, there is something to find! 

After the largely unsuccessful attempt of geneticists using candidate gene studies to identify genetic sequence variation that influences traits (REF), a new study design was proposed using arrays to measure hundreds of thousands of genetic variants across the genome in a hypothesis-free approach to identify sites, GWAS. Initially, these studies were conducted in hundreds or thousands of individuals and were identifying very few variants that could be said to reliably have an effect on the trait and these sites explained an extremely small proportion of the heriability estimates (<1%) (REF). Speculations were made about the reasons why this could be, for example the arrays were only capturing common genetic variation and it was rare genetic variation having the majority of the influence on phenotypes (REF). This provided a need to derive individual aspects of $h^2$ to inform future study design. If indeed common genetic variation contributed little to $h^2$ for the majority of traits, the GWAS approach would need to be re-thought and technologies would need to be made to capture genetic variants not reliably captured by existing arrays (e.g. rare variants). To this end, Yang et al. developed a method to estimate the contribution of all the single nucleotide polymorphisms (SNPs) to phenotypic variance (SNP-heriability or $h^2_{SNP}$) (REF). SNP-heriability was subsequently shown not to be inconsequential for complex traits and so warranted continuing use of GWAS and acquisition of larger samples to conduct these GWAS.

Lack of associations and lack of phenotypic variance captured by DNA methylation sites has largely been the case in EWAS, yet currently there is no corralary test to enable increased understanding of why this may be. Of course, there could be a variety of reasons for lack of signal in EWAS, which include what was discussed earlier, e.g. measuring the wrong tissue, but lots of small associations across the entire genome is also a possibility. In __Chapter \@ref(m2)__, I apply methods developed to assess SNP-heritability to estimate the proportion of phenotypic variance correlated with DNA methylation across a range of phenotypes, which can help inform future EWAS designs, like SNP-heritability studies helped influence the trajectories of genetic epidemiology. 

### Inferring biology from signals {#inferring-biology-from-signals}

A common goal of GWAS and EWAS is to understand the underlying biology of complex traits. Several techniques are often applied post-GWAS (REF) to attempt to accomplish this for genetic variants identified. A very common method is to map sites identified to genes and assess whether these genes are enriched for any particular biological pathway defined by various ontologies (geneset enrichment analysis), for example the Gene Ontology (GO) (REF) or KEGG (REF). This method has also been adapted to EWAS and applied several times (REFs). 

In geneset enrichment analysis, the signal of interest is mapped to a gene or genes and these are mapped to pathways. Then using some statistical test, such as the hypergeometric test or Fisher's exact test, evidence that the gene(s) are present in the pathways more than expected by chance is estimated. The null distribution from which the observed numbers are tested against, is usually estimated from the total number of genes tagged by the array of choice or by performing permutation tests.

As discussed previously in \@ref(dnam-phs) and in \@ref(mr), establishing causality from DNA methylation signal is difficult. Thus, when applying pathway enrichment analyses to EWAS signals, the pathways identified may actually be related to a particular confounder rather than the trait of interest. However, it is still of interest to perform the analysis because 1. it may give evidence that an EWAS signal is true (for example if an EWAS of diabetes picked up insulin pathways) and 2. it can provide some evidence for pathways involved in a trait if corroborated by other forms of evidence. 

Further, as DNA methylation changes may be as a result of the trait (unlike genetic variation), pathway analysis for EWAS studies could capture biological consequences of the trait. In __Chapter \@ref()__ I compare overlap of GWAS and EWAS signals in this context and discuss whether both study designs are likely to be useful in discovering the entire underlying biology of complex traits.

### Establishing causality {#establishing-causality}

As discussed, population-based studies of DNA methylation suffer from the same limitations as any observational epidemiology study, namely confounding and reverse causation. One method that aims to mitigate these limitations is Mendelian randomization (MR) (REFs). MR uses genetic variants as proxies for the exposure and outcome of interest in an instrumental variable framework (illustrated in __Figure \@ref(fig:)__). Using genetic variants as proxies ensures there is no reverse causation, as the variants are inherited, and reduces chances of confounding as individual genetic variants tend not to be associated with a broad range of phenotypes (REF). Furthermore, genetic variants are randomly segregated at conception, so in the abscence of assortative mating and dynastic effects, should be associated randomly with respect to any confounders. BLAH -- Need to read some MR papers again to get the wording right! 

TWO SAMPLE MR NEEDS EXPLAINING HERE

PLEIOTROPY NEEDS EXPLAINING HERE

MR can be applied to studies of DNA methylation by using methylation quantitative trait loci (mQTL), genetic variants associated with changes in DNA methylation levels, as proxies. By utilizing large trait GWAS and publically available mQTL databases (REFs) in a two-sample MR framework, good statistical power can be attained to estimate the effect of DNA methylation changes on various traits. Unfortunately, for each DNA methylation site few independent mQTLs have been identified, and of these trans-mQTLs (mQTLs over 1Mb from the DNA methylation site), are thought to have a higher probablity of being pleiotropic. This means that ruling out any associations that have arisen because of pleiotropy is difficult. However, applying MR in this context can still add evidence to the observational analyses that have been conducted. In __Chapter \@ref(dnam-mr)__ I use this principle to investigate the effect of DNA methylation on lung cancer.



__Might be an idea to add detail of some of these methods into appendices (e.g. mathematical basis of snp-heritability)__

* Give a little intro on what problems this chapter focuses on
* Describe the GWAS catalog and what it has done for genetic epi and how this can be adapted for EWAS
* Describe and explain heritability (and how it can be applied to EWAS)
* Describe pathway analysis??
* Describe MR


## Overview of thesis aims

__BELOW IS TERRIBLE BUT HELPFUL AS IT GIVES STRUCTURE__

EWAS might be an effective study technique to help provide biomarkers for better diagnosis, prognosis and prediction of disease and traits and further it could provide insight into trait aetiology as well as the downstream effects of a trait. However, with the field in it's infancy, there has yet to be a comprehensive look at what information has been gained from EWAS, what information is still to gain from EWAS in it's current state and whether or not the sites identified in EWAS are likely to be causally related to traits. 

In Chapter 3 the aim is to produce a database that brings together all the information currently published (as of X-date) together along with new EWAS with full summary statistics. This vast database will then be explored in Chapter 4, giving a detailed picture of what sites throughout the genome have been discovered in EWAS and exploring why they might have been discovered. NEED TO ADD IN HOW!

After exploring the information already gained from EWAS, Chapter 5 explores the amount of information still to gain from EWAS. NEED TO ADD IN HOW!

The database from Chapter 3 will again be used to extract large EWAS datasets for Chapter 6, where potential biological information from these EWAS will be compared to that of GWAS from the same traits. NEED TO ADD IN HOW!

Finally as an initial means of assessing whether DNA methylation might be causally associated with traits of interest, Chapter 7 will aim to use MR to help infer if DNA methylation identified in EWAS of lung cancer are likely to be causing lung cancer.