# Properties of epigenome-wide association studies {#properties-of-ewas}

```{r call_source-04}
read_chunk("chunks/04-properties_of_ewas.R")
```

``` {r load-data-04, results="hide", message=FALSE}
```

``` {r study-data-setup, results="hide", message=FALSE}
```

``` {r traits-manhattan-setup, results = "hide", message=FALSE}
```

``` {r rsq-setup, results = "hide", message=FALSE}
```

``` {r batch-faulty-setup, results = "hide", message=FALSE}
```

``` {r replication-setup, results = "hide", message=FALSE}
```

``` {r cpg-characteristics-setup, results = "hide", message=FALSE}
```

## Abstract {#abstract-04}

## Introduction {#introduction-04}
Learning from the successes and mistakes we make is what drives forward development. Hundreds of epigenome-wide association studies (EWAS) have been conducted in the last 10-15 years, no cross-EWAS studies, comparing results across a large group of EWAS results has been performed. By exploring the patterns of association across a large group of EWAS, one can discover potential explanations for the results found, that may shed light on failings in the literature as well as shared epigenetic architectures across traits.

Since the inception of EWAS, it has become clear that batch effects and cellular heterogeneity can generate false positives and bias effect sizes [@Price2018; @Forest2018; @Jaffe2014]. However, there are examples of replication amongst EWAS results, [@Nano2017; @Kaushal2017; @Morris2017; @Hedman2017; @Braun2017; @Teschendorff2015; @Zeilinger2013] and further, use of triangulation can be used to bolster evidence that changes in DNA methylation estimated are unlikely due to bias. By way of an example, changes in DNA methylation at _AHRR_ have been replicated across multiple smoking EWAS [@Zeilinger2013; @Elliott2014; @Joehanes2016; @Bojesen2017] and as functional reaseach has implicated this gene in handling toxic substances found in tobacco smoke [@Zudaire2008], it seems unlikely these findings are chance occurances. 

The characteristics of the DNA methylome may also explain some EWAS findings. Heritability varies across DNA methylation sites [@VanDongen2014; @McRae2014; @Hannon2018], and so if genetic effects are driving associations, either through confounding or with DNA methylation as a mediator, one would expect heritable sites to be commonly identified in EWAS. Variance is also heterogenous across sites [@Garg2018]. Some studies have suggested removing sites with particularly low variability to increase power [@Meng2010; @Logue2017], but it is unclear whether this may actually remove important sites. Experimental studies have shown DNA methylation changes at different locations of the genome correlate with different regulatory functions. For example, an increase in DNA methylation at transcriptional start sites is correlated with a decrease in gene expression [@Jones2012; @Ando2019; @Deaton2011], but an increase in DNA methylation within a gene body shows the opposite association [@Wolf1984; @Hellman2007]. Thus, genomic location of DNA methylation sites is likely to influence their likelihood of association with a trait.

Understanding underlying reasons that drive EWAS results is essential for future study design. This may come in the form of proper consideration of potential biasing factors, or by selecting certain DNA methylation sites based on their specific characteristics. Further, understanding the characteristics of DNA methylation-trait associations can inform the design of future technologies aimed at measuring DNAm for EWAS. 

Also, by examining the commonalities of EWAS results, one has the potential to uncover links between traits that have not previously been made or to identify new potential mediating factors between traits.

In this study we first describe the data present in the EWAS Catalog before exploring various explanations for the findings.

\newpage

## Methods {#methods-04}

### Epigenome-wide association studies data 
X EWAS summary statistics were extracted from the EWAS Catalog [REF]. This includes X published EWAS, X EWAS performed in the Accessible Resource for Integrated Epigenomics Studies (ARIES) subsection of the Avon Longitudinal Study of Parents and Children (ALSPAC), which is described below and X EWAS performed using data from the gene expression omnibus (GEO) resource.

All EWAS have a sample size of greater than 100 and measured more than 100,000 DNA methylation sites genome-wide. 

If there are multiple EWAS of the same trait, the EWAS with the largest N was selected for analyses when wishing to assess EWAS results of unique traits.

### ARIES 
Data used to derive estimates for DNA methylation variability and average methylation levels was taken from the ARIES subsection of the ALSPAC cohort. Pregnant women resident in Avon, UK with expected dates of delivery 1st April 1991 to 31st December 1992 were invited to take part in the study. The initial number of pregnancies enrolled is 14,541 (for these at least one questionnaire has been returned or a “Children in Focus” clinic had been attended by 19/07/1999). Of these initial pregnancies, there was a total of 14,676 foetuses, resulting in 14,062 live births and 13,988 children who were alive at 1 year of age. Full details of the cohort has been published previously (REFS). 

### Description of catalog data {-#description-of-data}
Associations between DNA methylation and traits, unless otherwise stated, were extracted at P < 1x10^-7^. Each of the CpGs in the Catalog are annotated to genes, using data from the meffil R package (REF). Variance explained ($r^2$) was calculated from P values ($p$) and sample size ($n$) as follows

\begin{equation}
    r^2 = XXX
    (\#eq:r-squared)
\end{equation}

We identified traits for which r^2^ values might be inflated. For each EWAS the estimated r^2^ values were summed and these values were transformed to try and approximate a normal distribution. Then a z-test was performed to assess which sum of r^2^ values were greater than the mean sum of r^2^ values. 

### Identifying faulty probes
By far the most common method to measure DNA methylation across the studies in The EWAS Catalog is using the Illumina Infinium HumanMethylation450 Beadchip. Since its development, the array has been extensively characterised (REFs) and it was found that not all probes map just to the CpG they were designed to bind to. Some probes map to SNPs, others are non-specific and some are prone to cross-hybridisation (REFs). We assigned probes to be 'potentially faulty' if they were characterised as such by Zhou et al. (REF). 

### Replication
An association was deemed to be replicated at P < 1x10^-4^. We assessed replicability of EWAS within the database in two separate ways. Firstly, replication within studies is recorded in the EWAS Catalog, thus simply performed a lookup for any studies that performed a replication or meta-analysed discovery and replication datasets. Secondly, we performed a lookup of results for any traits for which separarte multiple EWAS had been conducted. 

The Catalog also contains results from studies that have uploaded their data to GEO as well as results from the re-analysis of that data performed by us. These re-analyses adjusted for 20 surrogate variables only as many studies did not provide a complete set of covariates to GEO. We performed a lookup of results found in the original EWAS in the re-analysed data provided by GEO. 

### Selecting data to assess DNA methylation characteristics
We decided to only use studies and results thought to be more "robust" when assessing how characteristics of DNA methylation might impact EWAS results. To this end, we removed all potentially faulty probes and probes that mapped to sex chromomsomes, excluded studies with likely inflated r^2^ values, and removed studies for which re-analysis of the data replicated less than 10% of the findings.

### DNA methylation characteristics
The association between heritability, variability and average level of DNA methylation at each CpG site and EWAS effect size was assessed. To allow this across traits, we standardised beta coefficients, $\beta$ like so, 

\begin{equation}
    \beta_{standard} = ...
    (\#eq:standardised-beta-coeffs)
\end{equation}

Data on the variance and mean levels of DNA methylation across sites was provided by the Genetics of DNA methylation Consortium (GoDMC) (REF). Heritability of DNA methylation at each site has been previously estimated by X and Y, these values were kindly made publically available by the authors of those studies and were used in this study. 

Associations between each characteristic and effect size was assessed using linear regression, fitting the standardised effect size as the dependent variable and the characteristic as the independent variable. The standardised effect sizes were rank normalised to ensure normality.

### Enrichment tests
It was assessed whether the sites identified by EWAS that were present in the catalog were enriched in any genomic region etc. This analysis was performed using LOLA... 


### Code availability {#code-availability-04}
Code used to run the analyses is available here: https://github.com/thomasbattram/something

All analyses were completed using R (version __X__).

\newpage

## Results {#results-04}

### Description of the catalog {-#catalog-description}
Before assessing what might be underlying various EWAS results, we present a brief summary of the data in the EWAS Catalog (__Table \@ref(tab:study-data-tab)__).

```{r study-data-tab, message=FALSE, results="asis"}
```

The number of traits each CpG associated with was fairly even across chromosomes (__Figure \@ref(fig:traits-manhattan)__). There were `r num_to_text(nrow(h_genes))` CpGs that associated with more than ten traits, `r h_cpg_genes`. `r h_cpgs[length(h_cpgs)]` _`r names(h_cpgs)[length(h_cpgs)]`_ associated with more traits than any other site - `r max(h_genes$n_traits)` traits. These correspond mostly to metabolites, weight-related traits, and type two diabetes. 

(ref:traits-manhattan-cap) __Number of unique traits associated with each DNA methylation at each CpG.__ Sites associated with more than 10 unique traits are shown in orange and labelled.

```{r traits-manhattan, message=FALSE, results="asis", fig.cap='(ref:traits-manhattan-cap)'}
```

The total trait variance correlated with DNA methylation (r^2^) at each site varied from `r comma(min_r2)` to `r comma(max_r2)` (__Figure \@ref(fig:rsq-distribution)__). The sum of r^2^ values ranged greatly from `r comma(min(rsq_sum_dat$total_rsq))` to `r comma(max(rsq_sum_dat$total_rsq))` (__Figure \@ref(fig:rsq-sum-distribution)__), with a median of `r comma(median(rsq_sum_dat$total_rsq))`. There was evidence that `r num_to_text(nrow(suspect_studies))` studies had a total sum of r^2^ values greater than the mean (FDR < 0.05) and results from these studies made up the majority of r^2^ values greater than 0.1 (__Figure \@ref(fig:rsq-distribution)__).

(ref:rsq-distribution-cap) __Distribution of r-squared values across all CpG sites in The EWAS Catalog__. Each EWAS can identify multiple differentially methylated positions, each of which will capture some variance of the trait of interest for that EWAS (r^2^). $\sum {r^2}$ is the sum of r^2^ values, the distribution of which is shown in __Figure \@ref(fig:rsq-sum-distribution)__. `r num_to_text(nrow(suspect_studies), start_of_sentence = TRUE)` studies were identified for which there was strong evidence that the sum of r^2^ values were greater than the mean across all studies. All of the differentially methylated positions identified by those studies are highlighted in blue on the plot.

```{r rsq-distribution, message=FALSE, results="asis", fig.cap='(ref:rsq-distribution-cap)'}
```

(ref:rsq-sum-distribution-cap) __Distribution of the sum of r-squared values across each study in The EWAS Catalog.__ 

```{r rsq-sum-distribution, message=FALSE, results="asis", fig.cap='(ref:rsq-sum-distribution-cap)'}
```

### Robustness of results {#robustness-of-results}

`r sum(batch_tab$batch_included)` studies adjusted for batch effects in at least one model. Of all DMPs identifed, `r comma(percent_faulty_probes)`% were measured by potentially faulty probes (REF) and an extra `r comma(percent_sex_probes)`% were present on sex chromosomes (__Figure \@ref(fig:faulty-probes-plot)__).

(ref:faulty-probes-cap) __The percentage of differentially methylated positions that may have been identified by faulty probes and the percentage of EWAS that reported identifying at least one of these probes.__ Some CpGs are both on a sex chromosome and were identified as faulty by Zhou et al. They were labelled as 'potentially faulty'.

```{r faulty-probes-plot, message=FALSE, results="asis", fig.cap='(ref:faulty-probes-cap)'}
```

There were `r rep_study_n[rep_study_n$study == "discovery_and_replication", "n", drop=T]` studies that performed a meta-analysis of discovery and replication samples. A further `r rep_study_n[rep_study_n$study == "replication", "n", drop=T]` studies performed a separate replication analysis. Together, this provides `r nrow(replication_dat$rep_assoc)` associations within the EWAS Catalog that have been replicated at P < 1x10^-4^. 

From the studies that put their data on GEO, we re-analysed the association between DNA methylation and the phenotype of interest from the original study, including 20 surrogate variables as covariates. Both the original study results and the results from the re-analysis of the phenotype of interest are in The EWAS Catalog database for `r nrow(geo_rean)` studies. Across the studies, between `r min(geo_rean$rep_percent)`% and `r max(geo_rean$rep_percent)`% of DMPs were replicated at P < 1x10^-4^ (__Table \@ref(tab:replication-tab)__).

```{r replication-tab, message=FALSE, results="asis"}
```

Using the Catalog data we further performed replication analyses. There were `r nrow(nrow(rep_rates))` studies that shared a common phenotype of interest. Replication rate, judged as any reported CpG with P value < 1x10^-4^ in another study of the same trait, varied from `r comma(min(rep_rates$prop_rep))` to `r comma(max(rep_rates$prop_rep))` between studies (__Table \@ref(tab:geo-reanalysis-tab)__).  

```{r geo-reanalysis-tab, message=FALSE, results="asis"}
```

Before continuing to assess what CpG characteristics might, in part, explain some associations found in EWAS, we removed sites that were identified by a potentially faulty probe and were on either of the sex chromosomes. Further, we removed studies that had an inflated summed r^2^ value and studies for which less than 10% of sites identified in the original analyses were identified in a re-analysis using the data provided via GEO. Overall, this left `r rob_summary$n_ewas` EWAS and `r rob_summary$n_res` associations (at P < 1x10^-4^).

<!-- Smoking is associated with large changes in DNA methylation across the genome (REF) and is associated with many different traits (REF). Thus, it may confound DNA methylation associations found in the catalog. If this was the case, one might expect smoking related CpGs to appear more in the catalog than expected by chance. The DMPs identified by EWAS of traits other than smoking were enriched for smoking related CpG sites (P = X).
 -->
<!-- ### Correlation across tissues {-#correlation-across-tissues}

* There is correlation between DNA methylation sites across tissues, suggesting stability in DNAm, but would we expect to find associations at these positions across tissues?

* Replication of sites across tissues for same traits? 
* Do sites that are highly correlated across tissues appear more than expected by chance in whole blood EWAS? 
  + If yes then suggests the correlation might be due to decreased measurement error at those sites
  + If the opposite (correlated sites appear less than expected by chance), then it suggests that correlation occurs at positions that don't really matter (i.e. are just stable because of things like being at housekeeping genes)
* Can check housekeeping gene theory if needed
 -->

### CpG characteristics {#cpg-characteristics}

Using the EWAS results thought to be more robust, we investigated whether the characteristics of DNA methylation at CpG sites explained associations found in EWAS. 

Linear regression models gave some evidence that effect size associated with DNA methylation variance, the average DNA methylation level and heritability (P < 0.01, __Figure \@ref(fig:cpg-chars-plot)__). However, each of these CpG characteristics explained little of the variance across effect sizes (r^2^ < 0.001, __Figure \@ref(fig:cpg-chars-plot)__). 

(ref:cpg-chars-plot-cap) __Relationship between effect estimates of DNA methylation-trait associations and A. DNA methylation variability, B. mean methylation level, C. heritability.__ Effect estimates were scaled to represent the change in DNA methylation after an increase by one standard deviation of the trait of interest. Before assessing the association between effect estimates and DNA methylation characteristics, the standardised effect estimates were rank transformed to ensure normality. The table shows the strength of associations and the variance explained in effect estimates by each of the DNA methylation characteristics.

```{r cpg-chars-plot, message=FALSE, results="asis", fig.cap='(ref:cpg-chars-plot-cap)'}
```

* Enrichment of DMPs in various genomic regions etc.

\newpage

## Discussion {#discussion-04}

Understanding the nature of EWAS associations is imperative for biological inference. Using data from the EWAS Catalog we show that many CpGs associate with multiple different unique traits and effect sizes of these CpGs are inversely correlated with the variance of DNA methlyation. Roughly 10% of the differentially methylated positions identified were measured by potentially faulty probes (REF) and only X% of associations could be replicated.

### Identifying mediators {#identifying-mediators}

* X CpGs associated with multiple traits. One associated with X more than any other - X. 
* Traits it was associated with were weight-related, metabolites, T2D
* The association was replicated across some metabs + weight-related traits
* The gene it maps to is known to play a role in X
* CHECK MOST RECENT EWAS TO SEE IF THEY TEST ANY MEDIATING STUFF
* Highlights potential to use these databases for discovering important pathways between traits

### False positives {#false-positives}

* Databases can also highlight likely flaws. 
* So many traits with r2 > 1. Suggests, inflated effect estimates, redundance or both. 
* There were X studies that attempted to replicate results
* Replication rates varied a fair bit and many sites did not replicate
* Further, the replication rate using data from the same study was not high
* As with all study types, replication is a great way to reduce false positives

### Understanding CpG charactersitcs {#understanding-cpg-characteristics}

* Inverse correlation between effect size and variance likely reflect biological importance of stability for some sites. Same way that variation of certain phenotypes is limited by the essential body functions, like height.
* Effect sizes were fairly consistent across mean methylation levels at CpG sites. Might expect intermediate meth levels to show smaller effect sizes.
* h2 also fairly consitent across effect sizes, suggests that genetic effects aren't driving associations?

### Limitations {#limitations}

* Selection of studies
* Full summary stats not available for most studies

## Conclusion {#conclusion-04}
Overall, this manuscript demonstrates the potential for using large-scale EWAS databases to understand DNA methylation-trait associations. Unsurprisngly, we found that study design flaws can help explain some associations. However, it is noteworthy that the vast majority of studies have taken head of potential biasing factors, for example X% of studies adjusted for batch effects. Further, we found an invese association between DNA methylation variability and effect size, suggesting that studies that remove variable sites prior to analysis could be excluding important regions from the analysis. Finally, we identified ABCG1 as associated with X linked traits, which we think warrants follow-up analyses. This highlights the potential to use the EWAS Catalog to identify CpG sites of particular importance to various traits.

