# Discussion {#discussion-thesis}

## Overview of findings {#overview-08}

Phenotypic change cannot occur without molecular change. Identifying these changes adds to the aetiological understanding of traits and thus has the potential to uncover novel therapeutic targets (REFs). Further, studying the association between molecular marks and complex traits may yield new, valid predictors that could augment current prediction capacity within the clinic (REFs). 

Given the strong links made by experimental researchers between DNA methylation and the regulatory processes in cells (REFs), there was precedent for putting time and effort into studying whether DNA methylation and complex traits covary. 

Over the past 10-15 years, there have been some EWAS that have identified several reliable associations between DNA methylation and complex traits, for example with smoking (REFs) and body mass index (REFs). However, the trend has been for EWAS to identify few sites reliably associate with complex traits (REF). This was showcased in __Chapter \@ref(ewas-catalog)__. Understanding the aspects of EWAS that have led to this trend is imperative to progressing study designs and was the focus of this thesis.

In this body of work hundreds of published EWAS along with hundreds more EWAS I conducted were catalogued in an open access database. Using this data, I showed associations reported in EWAS to date are partly the result of unaccounted for technical effects, such as inclusion of poor quality probes, and characteristics of DNA methylation. Further, the variance captured by individual DNA methylation sites was highly varied and likely inflated amongst some studies. I further explored this covariation. It was found that the variance captured by all DNA methylation sites used in EWAS for 400 complex traits tended to be near zero, giving DNA methylation low total predicitive capacity across these traits. The capacity to understand aetiological aspects of traits from EWAS also appeared to be limited, although this was only tested for XXX traits. Upon comparison of associations between DNA methylation and lung cancer from observational EWAS and Mendelian randomization analyses, confounding was confirmed to be a problem, potentially accounting for a substantial proportion of observed EWAS associations. 

These findings suggest that substantial changes are required in the design of EWAS to aid the identification of valid predictors and enhance aetiological insight. 


* A brief look at smoking - features in all chapters of thesis, because of strong associations with DNAm. 

## Extensions to this work {#extensions-to-work}

### Beyond the HM450 in blood {#beyond-the-hm450}

This work focuses on the measures taken for contemporary EWAS, namely use of the HM450 to measure blood-based DNA methylation. Given the correlations structure across all DNA methylation sites in the genome is not known, this work can not be generalised to DNA methylation as a whole. Further, although certain aspects of observational EWAS, such as propensity for confounding, will always hinder interpretation of results, other aspects of this thesis may not apply to data from other tissues. 

As DNA methylation is so tightly linked to gene expression regulation, it seems likely that either DNA methylation or something coupled to it has a substantial impact on the aetiology of many complex traits. Thus, it seems likely the issues uncovered with EWAS in this thesis, such as the lack of covariation between DNA methylation and complex traits, is likely due to the fact too few sites are being measured in the wrong tissues. 

The impact of different methylation states of the genome, such as hydroxymethylation, is also relatively unknown (REFs). The current methods, which revolve around bisulphite conversion, capture all methylation states. If these states have differing effects then putting them under one umbrella will bias effect estimates. 

In a similar vein to other omics, sequencing technology will likely continue to improve and will become cheap enough to use across population samples. Also, despite the vast majority of EWAS being conducted using blood (__Table \@ref(tab:study-data-tab)__), there are XXX other tissue types recorded in the EWAS Catalog. The Cancer Genome Atlas (TCGA) has recorded DNA methylation data in a large number of tissues, and the trend to measure other tissues is likely to continue, especially as the number of reliable sites identified in EWAS tends to be low. In addition, manuscripts using techniques to measure gene expression and DNA methylation in single cells are becoming more common (REFs). Given the issues of cell specificity, these studies may prove pivotal in advancement of the epigenetic epidemiology field. 

As discussed in __Chapter \@ref(properties-of-ewas)__, DNA methylation is a binary measure, yet by pooling groups of cells and measuring methylation at each site as a proportion of the DNA molecules that are methylated, a continuous measure is derived. If cells of the same type are identical then one would expect they would be methylated at the same sites as one another. However, a mosaicism of methylation across cells may exist. If it is possible, then pooling groups of cells for analysis, and taking account differences between high order cell types, will remain a valid approach for EWAS. However, if strong evidence is provided that this cellular mosaicism is improbable between cells of the same type then rapid movement towards single-cell techniques is likely.

### New methods and data {#new-methods-and-data}

There is still substantial interest in assessing the association between DNA methylation and complex traits. In 2020 alone, there have been over XXX EWAS conducted that are suitable for adding to the EWAS Catalog. Further, new methods are being developed to measure the association between DNA methylation and complex traits. The group that developed GCTA, have also developed a command line tool to conduct EWAS using different models, OSCA (REF). There was recently a study that developed a Bayesian approach to conduct EWAS, using various confounders including genotype in their model (REF). These are just two examples of the many new methods that have been developed recently (REFs). 



* Complexity requires MR, other causal inference methods and a cross-discipline approach likely to be key to elucidating the impact of DNA methylation to complex traits (maybe mention colocolization here too)

* GoDMC provide huge dataset for estimating causal effects using MR

* Mass MR in GoDMC also backs up conclusions from this thesis - limited aetiological capacity for DNAm measured by the 450k! Blood QTLs may be limiting though - tissue specific QTLs found and needed to expand causal inference efforts

* Larger individual datasets, such as Generation Scotland could also be used to more precisely estimate the total predicitive capacity of DNAm for a particular trait and provide groups of traits for which DNAm may capture a substantial portion of variance

### Beyond DNA methylation {#beyond-dnam}

* Discussed why DNAm chosen to study (ref intro), but may not be ideal for many questions

* If DNAm not causal for many diseases, then there is little value for modifying it to prevent/treat disease is. May be worth looking to other epigenetic marks, but much work needed there!!

* Large efforts also started to quantify proteins. As the ultimate target for many pharmaceuticals, as understanding of function and quantification of proteins and their variants increases, protein-based epidemiological studies may supersede EWAS. They may also complement EWAS! 

* Still lots of problems faced by other molecular studies - especially tissue specificity! Also have added problem with gene expression + protein expression studies that you can have post-transcriptional and post-translational modifications that can alter gene/protein function...

## Final conclusions {#final-conclusions}

* Things are complex!! 

* EWAS have not yielded aetiological insights or the predictive capacity hoped in epidemiology. However, the constraints of technologies and samples have likely contributed a great deal to this. 

* This thesis provides evidence that the study design of current of EWAS is unlikely to provide substantial improvements in the ability to predict or understand the aetiological underpinnings of complex traits. 

* Key to understanding use of DNAm to complex trait aetiology will be diversification of tissue types, measurement of more sites, and experimental studies to confirm epidemiological inferences.

* Second stage of EWAS should take the full complexity of cellular and molecular signals into account. 


