# PhD project meeting notes 2020-07-23

Attendees: Tom B, Gib, Tom G, Nic

## Agenda

* Thesis intro comments
* Current timeline plan

## Thesis intro comments

### Overarching comments

* Many comments were specific and easy to change which is nice. I completely agree about restructuring the section discussing variance captured by all DNA methylation sites measured genome-wide -- need to explain what the problem is in DNA methylation before jumping to heritability measurements. 

* The other overarching comment was: "Got to here (end of inferring biology from signals section) and there has been a lot of discussion about a broad set of issues pertinent to EWAS and methylation – I have kid of forgotten what the job of the thesis is though – how does this all knit together?" -- This is something I really struggled with when writing the intro, knitting things together... 
	+ To remedy this just write something at the start of the "Using methods from genetics to help inform future EWAS section" that states something along the lines of 'In order to remedy some of the problems EWAS face and to help understand whether the "experiment" of measuring DNA methylation across many cohorts and studies has been successful, we can borrow ideas and methods developed in genetics and genetic epidemiology.' 
	+ Also need to add: 'There is no reason to believe that genotype and DNA methylation measurements share the same properties. They are distinct entities and it might be helpful to liken DNA methylation to more common phenotypic measurements rather than genotypic measurements. That being said, there are specific methods and ideas from genetics that can be adapted to explore epigenetics.'
	+ Then give examples as per TG's comment.



Would be worth talking more philosophically about the work. Why were the probes for 450k used? Was the "experiment" of measuring DNA methylation for the 450k in blood worth while or has everyone wasted their money? DNAm thought to be important enough to create multiple 'chips', but why? Are epigenetic signals surprising? There are an abundance of signal for certain phenotypes with low sample size, but of course you'd expect that if you liken DNAm to a phenotype rather than to a genotype

Things are written as if DNAm comes before changes in trait. State explicitly, early on in the intro, that DNAm changes can come after or before trait changes

Also worth discussing in more detail that there is experimental evidence that DNAm does not come before gene silencing in many cases.

Change wording from DNAm 'capturing' trait variance to DNAm and traits covarying! 

### Specific comments

* See intro doc --> Highlighted uncertain bits 

__NOTES:__

Some meeting notes

## Current timeline plan

### 2020-07-20 - 2020-07-26

* Intro comments attended to and re-written 
* ewas-gwas comp draft (Chapter 6) written 

### 2020-07-27 - 2020-08-02

* Intro checked over and sent back
* ewas-gwas comp draft finished and sent

### 2020-08-03 - 2020-08-09

* Finished extracting EWAS Catalog papers
* Finished properties of ewas (Chapter 4) analysis (analysis plan already agreed upon and fairly simple to run)

### 2020-08-10 - 2020-08-16

* Properties of ewas first draft written and sent off
* Started discussion

### 2020-08-17 - 2020-08-23

* Finished discussion and sent 
* Finished any outstanding results work and/or attended to comments

__NOTES:__

Some meeting notes

## AOB

* Send analysis plan to PY and MS for "properties of EWAS paper"