<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 1 Introduction | thesis.utf8</title>
  <meta name="description" content="" />
  <meta name="generator" content="bookdown 0.18.1 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 1 Introduction | thesis.utf8" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 1 Introduction | thesis.utf8" />
  
  
  




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="index.html"/>
<link rel="next" href="2-methods.html"/>
<script src="libs/header-attrs-2.1.1/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="1-introduction.html"><a href="1-introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="1-introduction.html"><a href="1-introduction.html#dna-methylation-as-part-of-the-regulatory-machinery"><i class="fa fa-check"></i><b>1.1</b> DNA methylation as part of the regulatory machinery</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="1-introduction.html"><a href="1-introduction.html#gene-expression-regulation"><i class="fa fa-check"></i><b>1.1.1</b> Gene expression regulation</a></li>
<li class="chapter" data-level="1.1.2" data-path="1-introduction.html"><a href="1-introduction.html#defining-epigenetics"><i class="fa fa-check"></i><b>1.1.2</b> Defining epigenetics</a></li>
<li class="chapter" data-level="1.1.3" data-path="1-introduction.html"><a href="1-introduction.html#histone-modifications"><i class="fa fa-check"></i><b>1.1.3</b> Histone modifications</a></li>
<li class="chapter" data-level="1.1.4" data-path="1-introduction.html"><a href="1-introduction.html#dna-methylation"><i class="fa fa-check"></i><b>1.1.4</b> DNA methylation</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-introduction.html"><a href="1-introduction.html#dnam-phs"><i class="fa fa-check"></i><b>1.2</b> Use of DNA methylation in a population settingÂ </a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="1-introduction.html"><a href="1-introduction.html#appeal-of-dnam"><i class="fa fa-check"></i><b>1.2.1</b> The appeal of DNA methylation to epidemiologists</a></li>
<li class="chapter" data-level="1.2.2" data-path="1-introduction.html"><a href="1-introduction.html#ewas"><i class="fa fa-check"></i><b>1.2.2</b> Epigenome-wide association studies</a></li>
<li class="chapter" data-level="1.2.3" data-path="1-introduction.html"><a href="1-introduction.html#problems-for-ewas"><i class="fa fa-check"></i><b>1.2.3</b> Problems for EWAS</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="1-introduction.html"><a href="1-introduction.html#genetics-in-ewas"><i class="fa fa-check"></i><b>1.3</b> Using methods from genetics to help inform future EWAS</a>
<ul>
<li class="chapter" data-level="1.3.1" data-path="1-introduction.html"><a href="1-introduction.html#gwas-catalog"><i class="fa fa-check"></i><b>1.3.1</b> The GWAS Catalog</a></li>
<li class="chapter" data-level="1.3.2" data-path="1-introduction.html"><a href="1-introduction.html#heritability"><i class="fa fa-check"></i><b>1.3.2</b> Total variance captured by all sites measured genome-wide</a></li>
<li class="chapter" data-level="1.3.3" data-path="1-introduction.html"><a href="1-introduction.html#inferring-biology-from-signals"><i class="fa fa-check"></i><b>1.3.3</b> Inferring biology from signals</a></li>
<li class="chapter" data-level="1.3.4" data-path="1-introduction.html"><a href="1-introduction.html#establishing-causality"><i class="fa fa-check"></i><b>1.3.4</b> Establishing causality</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="1-introduction.html"><a href="1-introduction.html#overview-of-thesis-aims"><i class="fa fa-check"></i><b>1.4</b> Overview of thesis aims</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-methods.html"><a href="2-methods.html"><i class="fa fa-check"></i><b>2</b> Methods</a></li>
<li class="chapter" data-level="3" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html"><i class="fa fa-check"></i><b>3</b> The EWAS Catalog: a database of epigenome-wide association studies</a>
<ul>
<li class="chapter" data-level="3.1" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#abstract"><i class="fa fa-check"></i><b>3.1</b> Abstract</a></li>
<li class="chapter" data-level="3.2" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#introduction-1"><i class="fa fa-check"></i><b>3.2</b> Introduction</a></li>
<li class="chapter" data-level="3.3" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#methods-1"><i class="fa fa-check"></i><b>3.3</b> Methods</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#implementation"><i class="fa fa-check"></i><b>3.3.1</b> Implementation</a></li>
<li class="chapter" data-level="3.3.2" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#overview-of-publication-data-extraction"><i class="fa fa-check"></i><b>3.3.2</b> Overview of publication data extraction</a></li>
<li class="chapter" data-level="3.3.3" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#overview-of-geo-data-extraction"><i class="fa fa-check"></i><b>3.3.3</b> Overview of GEO data extraction</a></li>
<li class="chapter" data-level="3.3.4" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#ewas-methods"><i class="fa fa-check"></i><b>3.3.4</b> EWAS methods</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#results"><i class="fa fa-check"></i><b>3.4</b> Results</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#database-interface-and-use"><i class="fa fa-check"></i><b>3.4.1</b> Database interface and use</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="3-ewas-catalog.html"><a href="3-ewas-catalog.html#discussion"><i class="fa fa-check"></i><b>3.5</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-properties-of-ewas.html"><a href="4-properties-of-ewas.html"><i class="fa fa-check"></i><b>4</b> Properties of EWAS</a></li>
<li class="chapter" data-level="5" data-path="5-h2ewas-chapter.html"><a href="5-h2ewas-chapter.html"><i class="fa fa-check"></i><b>5</b> h2ewas</a></li>
<li class="chapter" data-level="6" data-path="6-ewas-gwas-comp-chapter.html"><a href="6-ewas-gwas-comp-chapter.html"><i class="fa fa-check"></i><b>6</b> EWAS-GWAS comparison</a></li>
<li class="chapter" data-level="7" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html"><i class="fa fa-check"></i><b>7</b> Appraising the causal relevance of DNA methylation for risk of lung cancer</a>
<ul>
<li class="chapter" data-level="7.1" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#abstract-1"><i class="fa fa-check"></i><b>7.1</b> Abstract</a>
<ul>
<li class="chapter" data-level="7.1.1" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#background"><i class="fa fa-check"></i><b>7.1.1</b> Background</a></li>
<li class="chapter" data-level="7.1.2" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#methods-2"><i class="fa fa-check"></i><b>7.1.2</b> Methods</a></li>
<li class="chapter" data-level="7.1.3" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#results-1"><i class="fa fa-check"></i><b>7.1.3</b> Results</a></li>
<li class="chapter" data-level="7.1.4" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#conclusions"><i class="fa fa-check"></i><b>7.1.4</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#introduction-2"><i class="fa fa-check"></i><b>7.2</b> Introduction</a></li>
<li class="chapter" data-level="7.3" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#methods-3"><i class="fa fa-check"></i><b>7.3</b> Methods</a>
<ul>
<li class="chapter" data-level="7.3.1" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#ewas-study-details"><i class="fa fa-check"></i><b>7.3.1</b> EWAS study details</a></li>
<li class="chapter" data-level="7.3.2" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#ewas-meta-analysis"><i class="fa fa-check"></i><b>7.3.2</b> EWAS Meta-analysis</a></li>
<li class="chapter" data-level="7.3.3" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#mendelian-randomization-1"><i class="fa fa-check"></i><b>7.3.3</b> Mendelian randomization</a></li>
<li class="chapter" data-level="7.3.4" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#sample-2-transdisciplinary-research-in-cancer-of-the-lung-and-the-international-lung-cancer-consortium"><i class="fa fa-check"></i><b>7.3.4</b> Sample 2: Transdisciplinary Research in Cancer of the Lung and The International Lung Cancer Consortium</a></li>
<li class="chapter" data-level="7.3.5" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#supplementary-analyses"><i class="fa fa-check"></i><b>7.3.5</b> Supplementary analyses</a></li>
</ul></li>
<li class="chapter" data-level="7.4" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#results-2"><i class="fa fa-check"></i><b>7.4</b> Results</a>
<ul>
<li class="chapter" data-level="7.4.1" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#ewas-meta-analysis-1"><i class="fa fa-check"></i><b>7.4.1</b> EWAS meta-analysis</a></li>
<li class="chapter" data-level="7.4.2" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#mendelian-randomization-2"><i class="fa fa-check"></i><b>7.4.2</b> Mendelian randomization</a></li>
<li class="chapter" data-level="7.4.3" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#tumour-and-adjacent-normal-lung-tissue-methylation-patterns"><i class="fa fa-check"></i><b>7.4.3</b> Tumour and adjacent normal lung tissue methylation patterns</a></li>
<li class="chapter" data-level="7.4.4" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#gene-expression-associated-with-mqtls-in-blood-and-lung-tissue"><i class="fa fa-check"></i><b>7.4.4</b> Gene expression associated with mQTLs in blood and lung tissue</a></li>
</ul></li>
<li class="chapter" data-level="7.5" data-path="7-dnam-lung-cancer-mr.html"><a href="7-dnam-lung-cancer-mr.html#discussion-1"><i class="fa fa-check"></i><b>7.5</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="conclusion.html"><a href="conclusion.html"><i class="fa fa-check"></i>Conclusion</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-the-first-appendix.html"><a href="A-the-first-appendix.html"><i class="fa fa-check"></i><b>A</b> The First Appendix</a></li>
<li class="chapter" data-level="B" data-path="B-the-second-appendix-for-fun.html"><a href="B-the-second-appendix-for-fun.html"><i class="fa fa-check"></i><b>B</b> The Second Appendix, for Fun</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="introduction" class="section level1" number="1">
<h1><span class="header-section-number">Chapter 1</span> Introduction</h1>
<p>DNA methylation is a dynamic, chemical modification to DNA that can occur at millions of sites across the genome. The modification may impact various traits as well as be an important biomarker or predictor. This thesis will focus on the most common study design for assessing the association between DNA methylation and complex traits, epigenome-wide association studies (EWAS).</p>
<p>To help inform design of future studies, there is a need to understand what has been discovered thus far in EWAS and what future EWAS are likely to add in the context of current EWAS and other study designs. A key element currently not understood is why DNA methylation at certain sites or regions of the genome tends to be associated with multiple traits. Potential reasons include technical artefacts, genetic factors or other strong confounding factors that are biasing results, or they could genuinely be of biological importance. Further, the proportion of trait variation captured by DNA methylation at the sites measured is unknown, which means one can only speculate at how much information can be gained from EWAS. Even if there is variation captured by DNA methylation sites, itâs unclear whether identifying these sites provide extra biological understanding of traits above what can be found from other methods. It is also imperative to fully explore whether these sites are causally related to the trait of interest before making claims about how DNA methylation influences it.</p>
<p>In this chapter I describe DNA methylation in the context of the regulatory processes in human cells, discuss itâs potential for use in population level studies and describe the current state of EWAS research. Then I explain how epigenetic epidemiologists can draw on the methods developed by geneticists to understand 1. what information has been gained from EWAS, 2. what information is left to gain from EWAS and 3. the causal nature of DNA methylation-trait associations identified in EWAS.</p>
<div id="dna-methylation-as-part-of-the-regulatory-machinery" class="section level2" number="1.1">
<h2><span class="header-section-number">1.1</span> DNA methylation as part of the regulatory machinery</h2>
<p>DNA methylation is highly correlated with gene expression regulation <span class="citation">(<span class="citeproc-not-found" data-reference-id="Suzuki2008"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Siegfried1999"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Bird2002"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Jones2012"><strong>???</strong></span>)</span>. This is the main mechanism by which it is thought DNA methylation can influence changes in phenotype. However, DNA methylation is just one of many epigenetic marks that are involved in gene regulation (<strong>Figure <a href="#fig:epigenetic-marks"><strong>??</strong></a></strong>), so here I briefly outline some of these regulatory marks and discuss DNA methylation in the context of this complex biological process.</p>
<div id="gene-expression-regulation" class="section level3" number="1.1.1">
<h3><span class="header-section-number">1.1.1</span> Gene expression regulation</h3>
<p>Gene expression is tightly controlled and only in the right circumstances will RNA polymerase II succeed in binding the correct site, initiating and finally completing the transcription process <span class="citation">(<span class="citeproc-not-found" data-reference-id="Kornberg1999"><strong>???</strong></span>)</span>. The importance of this level of control is no more apparent than in the developmental stages of human life. Humans start as a single cell and after roughly nine months are transformed into a multicellular organism with trillions of cells, including hundreds of unique cell types. As these cells arise from a single progenitor they must contain identical genetic sequences (except a few somatic mutations). Therefore, the process by which the body is able to create such diversely functioning cells and tissues, must come from regulation of how the genetic sequence is read and from the regulation of its products. Indeed it was in developmental biology that we first began to understand the processes that may regulate gene expression (REFs - may have to remove this sentence as canât find an adequate referenceâ¦). There are a plethora of methods cells use to regulate gene and protein expression post-transcriptionally <span class="citation">(<span class="citeproc-not-found" data-reference-id="Hafner2019"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Corbett2018"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Wang2014"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Filipowicz2008"><strong>???</strong></span>)</span>, but these are beyond the scope of this thesis. So Iâll continue to describe gene expression regulation just in the context of epigenetic marks.</p>
</div>
<div id="defining-epigenetics" class="section level3" number="1.1.2">
<h3><span class="header-section-number">1.1.2</span> Defining epigenetics</h3>
<p>The definition of epigenetics is much debated <span class="citation">(<span class="citeproc-not-found" data-reference-id="Greally2018"><strong>???</strong></span>)</span>. In the 1940s the âepigenetic landscapeâ was introduced by Waddington to describe how genes influence cell fates <span class="citation">(<span class="citeproc-not-found" data-reference-id="Stern2000"><strong>???</strong></span>)</span>. Since then the term âepigeneticsâ has been used in many forms, so to avoid confusion, throughout the rest of the thesis, here is what I mean when using the term <strong>epigenetics</strong>: the study of mitotically heritable changes in gene expression that occur without changes in DNA sequence. When referring to <strong>epigenetic marks</strong> Iâm referring to chemical changes to the genome and genome-bound proteins that are mitotically heritable and may influence gene expression without changing the DNA sequence.</p>
</div>
<div id="histone-modifications" class="section level3" number="1.1.3">
<h3><span class="header-section-number">1.1.3</span> Histone modifications</h3>
<p>Histones are DNA-bound proteins that when modified can influence chromatin state and transcription factor interactions with the DNA. These proteins are contained in nucleosomes, which consist of four histone proteins, each of which is present twice in the nucleosome. Post-translational modifications can occur to any of the histone monomers and these have been associated with both positive and negative changes in gene expression <span class="citation">(<span class="citeproc-not-found" data-reference-id="Bannister2011"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Berger2007"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Jenuwein2001"><strong>???</strong></span>)</span>. Histone modifications are numerous and complex in nature. To briefly describe the complexity, there are at least ten types of histone modification that can occur, each of the histone monomers can be modified across many different sites, and for any one site mutiple of the same modification can occur. It is the combination of modifications across all histones that plays a role in gene expression regulation <span class="citation">(Bannister2011; <span class="citeproc-not-found" data-reference-id="Berger2007"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Jenuwein2001"><strong>???</strong></span>)</span>. Furthermore, histone modifications are subject to rapid change upon environmental stimulus to help induce or repress gene expression (REFs). Consequently, functions of only a few histone marks have been reliably established thus far (REFs) and taking a snapshot of the histone modifications present in cells may give a poor summary of how gene regulation is occurring. Very few population-based studies have assessed histone modifications because of these, plus some other, technical difficulties (REFs). They may become far more prominent in the future as our understanding and ability to measure the modifications in a meaningful way increases.</p>
</div>
<div id="dna-methylation" class="section level3" number="1.1.4">
<h3><span class="header-section-number">1.1.4</span> DNA methylation</h3>
<p>DNA methylation is the addition of a methyl group to the DNA, which primarily occurs at the 5â cytosine of a CpG site. Little is known about the role of non-CpG site DNA methylation and current EWAS tend to only measure CpG methylation, so that will be my focus here. The function of this epigenetic mark was proposed back in 1975, where two papers suggested it represses gene expression <span class="citation">(<span class="citeproc-not-found" data-reference-id="Holliday1975"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Riggs1975"><strong>???</strong></span>)</span>. Since then, its association with various important intracellular processes such as X-inactivation, genomic imprinting and suppression of transposon action have been elucidated <span class="citation">(<span class="citeproc-not-found" data-reference-id="Riggs1975"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Bell2000"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Yoder1997"><strong>???</strong></span>)</span>. Despite this, the role DNA methylation plays in regulating gene expression is not fully understood and research is still ongoing.</p>
<p>One thing ressearch has revealed is that the location of DNA methylation is important to its relationship with gene expression. CpG sites are not randomly distributed throughout the genome, but are often found in clusters <span class="citation">(<span class="citeproc-not-found" data-reference-id="Jones2012"><strong>???</strong></span>)</span> and itâs thought that methylation and de-methylation of CpG sites in groups is what drives their association with regulatory function. Location of the CpG sites relative to genes and other regulatory elements is also of importance. Several studies have shown that increased DNA methylation at transcription start sites tends to be associated with a decrease in gene expression (REFs), but gene body DNA methylation is positively correlated with expression (REFs) (<strong>Figure <a href="1-introduction.html#fig:dnam-functions">1.1</a></strong>). This suggests that regulation of DNA methtylation in clusters at specific sites relative to genes are what drive its relationship with gene expression. To support this, there are clear biological processes that regulate DNA methylation at nearby sites together, for example, CpGs at transcription factor binding sites can be de-methylated as a group when the transcription factor binds (REF). Further, nearby sites are often statistically correlated (REF). However, there is no evidence to suggest that neighbouring sites do indeed act in tandem or whether it is likely one site from the group is driving regulatory function. This is something I explore in (<strong>Chapter <a href="5-h2ewas-chapter.html#h2ewas-chapter">5</a></strong>).</p>
<p>These strong associations between DNA methylation and gene expression does not necessarily mean that the addition or removal of methyl groups will actively impact gene expression. Elucidating the causal nature of the assoction between DNA methylation and gene expression has been fraught with difficulties and has often provided conflicting results. One study knocked out an enzyme that catalyses the addition of methyl groups to the DNA, DNA methyltransferase X, in haematopoeitic stem cells and found these cells would not differentiate, suggesting gene expression changes required for differentiation were not possible without addition of methyl groups to the DNA (REF). However, a recent paper suggested that any DNA methylation changes associated with alterations in gene expression were just a by-product of other regulatory processes such as transcription factor binding (REF). To further complicate things, if DNA methylation does influence gene expression, the mechanism of action is unclear and may depend on the gene being examined. One study showed the presence of DNA methylation at the transcription start site of gene <em>X</em> prevented transcription factor binding (REF), but another showed a highly methylated transcription start site had no impact on the affinity a transcription factor had for that site (REF). Although the body of work presented in this thesis does not aim to explore if and how DNA methylation influences gene expression, it is important to note the relationship between the two isnât entirely clear when thinking of implications of population studies using DNA methylation, this will be discussed further in the following sections.</p>

<div class="figure" style="text-align: center"><span id="fig:dnam-functions"></span>
<img src="figure/01-introduction/dnam-gene-expression.pdf" alt="Simplified diagrams of the associations between DNA methylation and gene regulation. In a, CpG sites are methylated at the promoter region, but not in the gene body, which is associated with lack of gene expression. In b, the opposite is occurring. In c, the transposon is fully methylated and blocked from being transcribed."  />
<p class="caption">
Figure 1.1: Simplified diagrams of the associations between DNA methylation and gene regulation. In <em>a</em>, CpG sites are methylated at the promoter region, but not in the gene body, which is associated with lack of gene expression. In <em>b</em>, the opposite is occurring. In <em>c</em>, the transposon is fully methylated and blocked from being transcribed.
</p>
</div>
</div>
</div>
<div id="dnam-phs" class="section level2" number="1.2">
<h2><span class="header-section-number">1.2</span> Use of DNA methylation in a population settingÂ </h2>
<p>The importance of DNA methylation in disease has already been established in rare developmental disorders caused by aberrant imprinting patterns <span class="citation">(<span class="citeproc-not-found" data-reference-id="Wrzeska2004"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Nicholls2000"><strong>???</strong></span>)</span>, but the relationship between DNA methylation and complex phenotypes is unclear and is the focus of this thesis. In this section I discuss the appeal of studying DNA methylation in this context, introduce the most common study design for doing so, EWAS, and overview some successes and complications of the work.</p>
<div id="appeal-of-dnam" class="section level3" number="1.2.1">
<h3><span class="header-section-number">1.2.1</span> The appeal of DNA methylation to epidemiologists</h3>
<p>All epigenetic modifications are of potential interest to those studying any phenotype. Arguably, epigenetics could underlie all phenotypic changes and be the difference between individuals who develop disease and those who donât <span class="citation">(<span class="citeproc-not-found" data-reference-id="Relton2010"><strong>???</strong></span>)</span>. Further, epigenetic marks are modifiable, which means theoretically it would be possible to prevent or treat any disease by altering the epigenetic patterns of individuals <span class="citation">(<span class="citeproc-not-found" data-reference-id="Weaver2004"><strong>???</strong></span>)</span>. There are large practical issues with this, which are discussed in more detail in <a href="1-introduction.html#problems-for-ewas">Problems for EWAS</a>. However, even if targeting epigenetic marks is not easy, as long as it is possible to observe them they could be used as diagnostic biomarkers and predictors <span class="citation">(<span class="citeproc-not-found" data-reference-id="Relton2010"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Birney2016"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Koch2018"><strong>???</strong></span>; <span class="citeproc-not-found" data-reference-id="Hentze2019"><strong>???</strong></span>)</span>. Thus, the ability to measure, and the research in to understanding epigenetic mechanisms, could have broad consequences for public health.</p>
<p>A major difference between DNA methylation and other epigenetic marks, that was alluded to earlier, is that DNA methylation is far more stable. Enzymes do exist that can actively de-methylate the DNA, for example the ten-eleven translocation (TET) enzymes (REFs), but cell division or excision of the nucleotide is required for full de-methylation of a DNA molecule (Bhutani, N. et al 2010, Popp, C. et al.Â 2010). Biologically, this suggests DNA methylation might be involved in long-term repression of gene expression, which is thought to be the case for X-inactivation (REF), and practically it makes studying the epigenetic mark easier because stability ensures the marks are more resistent to changes after collection of samples and further, variation over a short time period across the lifecourse (e.g.Â over the span of hours or days) is likely to be minimal <strong>WOULD BE NICE TO HAVE A REFERENCE FOR THIS!!!</strong>. Also, even though it isnât clear that DNA methylation precedes gene expression regulation, the regulatory processes that govern whether genes are transcribed are linked. For example, there is an inverse correlation between DNA methylation of a transcription start site and transcription factor binding of that site (REF). Further, there are known examples of how DNA methylation tends to associate with other epigenetic marks, including positive correlation with the histone modification H3K9me3 <span class="citation">(<span class="citeproc-not-found" data-reference-id="Jones"><strong>???</strong></span>)</span> and histone deacetylation (Jones 2012, Wade, P. A. &amp; Wolffe, A. P 2001). This means measuring DNA methylation may capture regulatory information, even if addition or removal of these marks would have little impact on gene expression. Recently, itâs been shown that epigenetic marks can be used to predict each other with high accuracy (Ernst and Kellis 2015), adding weight to the argument DNA methylation can proxy for other regulatory elements.</p>
</div>
<div id="ewas" class="section level3" number="1.2.2">
<h3><span class="header-section-number">1.2.2</span> Epigenome-wide association studies</h3>
<p>EWAS are the most common study design for assessing the association between DNA methylation and a complex trait (REF). They typically involve measuring hundreds of thousands of DNA methylation sites across the genome in a case-control or cohort setting and using linear models to assess the association between DNA methylation and the trait of interest. Itâs a study design that has been widely adopted over the past ten years and the relationship between a plethora of traits, from smoking to anthropometric measures to childhood adversities, and DNA methylation has now been studied (REFs). There are also large consortia that are pooling samples to gain power for these studies, for example the Pregnancy and Epigenetics Consortium (PACE) (REF). A few traits have been identified as being associated with large variations in DNA methylation, one of which is smoking, where thousands of sites have been identified and many replicated in EWAS of smoking (REFs). DNA methylation can predict smoking status, potentially better than self-report (REF). It has been revealed the direction of effect at some sights is likely from smoking to DNA methylation and in fact over time the DNA methylation changes caused by smoking may be (mostly) reversible by giving up smoking (REF). Another complex trait shown to relate to large variation in DNA methylation across the genome is age (REFs), so much so that DNA methylation makes a highly accurate predictor for age (REFs). These studies have shown that large perturbations in the DNA methylome can be related to complex traits, highlighting the potential for EWAS in population studies.</p>
</div>
<div id="problems-for-ewas" class="section level3" number="1.2.3">
<h3><span class="header-section-number">1.2.3</span> Problems for EWAS</h3>
<p>Despite the promise of understanding the underlying biological processes related to traits, studying the relationship between DNA methylation and complex traits provides many practical difficulties that often make the results of EWAS hard to interpret (REF).</p>
<div id="confounding" class="section level4" number="1.2.3.1">
<h4><span class="header-section-number">1.2.3.1</span> Confounding</h4>
<p>One issue that plagues all observational epidemiology, including EWAS, is confounding. This is where the traits of interest share a common cause, and can lead to associations between traits despite a lack of causality. Complex traits are strongly correlated with each other, often in clusters, which can lead to large amounts of measured and unmeasured confounding being present in EWAS. Of course, in order to produce therapies to prevent or treat disease by altering DNA methylation or other parts of the epigenome, causality must be established. Therefore, problems of confounding must be overcome in EWAS to use these results to start developing methods of targetting DNA methylation changes, this is discussed more in <a href="1-introduction.html#establishing-causality">Establishing causality</a>.</p>
</div>
<div id="cell-type-heterogeneity" class="section level4" number="1.2.3.2">
<h4><span class="header-section-number">1.2.3.2</span> Cell type heterogeneity</h4>
<p>As discussed, epigenetic factors guide differentiation of a single pluripotent cell to hundreds of cell types in human development. As these cell types can have large differences in morphology and function, it is clear that epigenetic marks, including DNA methylation, will vary widely between cell types (REF). This poses two distinct problems for EWAS. Firstly, when collecting samples to measure DNA methylation, unless cells are purified, then a pool of cell types will be present in the samples, each with their own distinct DNA methylation patterns. This can lead to issues of confounding by cell type. For example, in a case control study, cases may be more likely to have increased numbers of CD4+ Th2 immune cells and these cells may on average have a higher level of DNA methylation at site X. In this scenario if one were to take blood cells, measure DNA methylation, and assess the association between DNA methylation and the trait of interest, one might find an association between DNA methylation at site X and the trait, but this is just a function of the increased number of CD4+ Th2 cells present in cases and site X has no causal relationship with the trait itself. There have been efforts to try and account for cell type heterogeneity in EWAS (REFs), but to completely prevent itâs confounding effects, cells should be purified. The second problem arising from cell-type specific patterns of DNA methylation is the uncertainty that the cell type being studied is in fact one in which DNA methylation covaries with the trait of interest. Non-invasive cells to collect, such as blood, skin, and saliva, are common amongst epidemiological studies, but it is unclear whether EWAS in these studies are relevant to a large proportion of complex traits. This is studied, with regards to blood, in <strong>Chapter <a href="5-h2ewas-chapter.html#h2ewas-chapter">5</a></strong>. Studies have actually shown high levels of correlation between DNA methylation of different cell types (REFs), but it is unknown whether the correlated sites are important to trait variation. Further, this correlation may complicate things with regards to epigenetic therapies. Some organs, such as the brain, are difficult to target with therapies (REF) so even if a promising epigenetic target is identified, it may be almost impossible to translate this into something clinically useful.</p>
</div>
<div id="measuring-dna-methylation" class="section level4" number="1.2.3.3">
<h4><span class="header-section-number">1.2.3.3</span> Measuring DNA methylation</h4>
<p>Ideally, in every sample, DNA methylation would be measured across all sites in the genome. Unfortunately, this is not currently possible and sequencing technologies that offer something similar are often very expensive (REFs). There are three alternatives available to DNA methylation studies. Firstly, one could sequence a small portion of the genome if this section is of particular interest, however this candidate gene approach is unlikely to be profitable unless the genes targeted already have good evidence for epigenetic variation with the trait of interest (REFs). Secondly, measuring DNA methylation on repeat sequences of the genome, such as long interspersed nuclear elements (LINEs) and short interspersed nuclear elements (SINEs), can provide estimates for global DNA methylation changes (REFs). This is useful to examine if a trait is related to large perturbations of DNA methylation across the genome, but gives little mechanistic insight into what effects these changes may be having, as methylation at functional genes is not measured. Thirdly, an array approach covering DNA methylation genome-wide at selected sites (REFs). This last approach is the most common for EWAS as it enables measurement of DNA methylation at hundreds of thousands of sites at a relatively cheap price per sample (REFs). The most commonly used array currently is the Illumina Infinium HumanMethylation450 (HM450) Beadchip, which measures DNA methylation at over 450,000 sites across the whole genome (REF). The technology is always improving and now the Illumina Infinium HumanMethylationEPIC Beadchip, measuring over 800,000 sites is being adopted by various studies (REFs). The arrays are designed to cover a large number of genes and have many probes in promoter regions (REF). With current biological knowledge of DNA methylation (see section <a href="1-introduction.html#dna-methylation">1.1.4</a>) this is unsurprising, but these arrays only measure roughly 1.5-4% of all CpG sites in the genome. Further, there is no empirical evidence to show these sites covary with complex traits. This will be examined in <strong>Chapter <a href="5-h2ewas-chapter.html#h2ewas-chapter">5</a></strong>.</p>
<p>Batch effects and technical artefacts can also substantially bias results in EWAS (REFs). There has been considerable effort made to try and mitigate these effects and methods have been developed to help correct for batch effects (REFs). Further, studies have used sequencing methods to identify potentially unreliable probes (REFs). Removal of batch effects is readily employed in EWAS, but it is unclear as to whether potentially faulty probes have given rise to an excess of EWAS signal. This information could provide more evidence of the technical issues facing EWAS and prompt extra care when conducting these studies. This is examined briefly in <strong>Chapter <a href="4-properties-of-ewas.html#properties-of-ewas">4</a></strong>.</p>
</div>
<div id="complexity-of-regulatory-mechanisms" class="section level4" number="1.2.3.4">
<h4><span class="header-section-number">1.2.3.4</span> Complexity of regulatory mechanisms</h4>
<p>EWAS identify single sites in the genome for which DNA methylation variation is associated with a trait of interest. As discussed, DNA methylation at a single site will likely be correlated with DNA methylation at neighbouring sites and other nearby epigenetic marks. This makes inferring mechanism of action very difficult. Differentially methylated region (DMR) analysis is often employed, which aims to determine if multiple neighbouring sites share an association with the trait of interest with the same direction of effect (REFs). These give evidence to whether the sites covary similarly with the trait of interest, but donât provide evidence that the sites are acting independently or not. A simple way to view EWAS, is that they identify potentially important regions of the genome with respect to the trait of interest, i.e.Â they suggest that genes in that region may be influenced by or influence the trait of interest. However, this simplification may only be appropriate for a limited number of sites in the genome and the complexity of regulatory processes make it difficult to understand the effect changes in DNA methylation at any one locus may have.</p>
</div>
<div id="treatments" class="section level4" number="1.2.3.5">
<h4><span class="header-section-number">1.2.3.5</span> Treatments</h4>
<p>This is not a focus of my thesis, but for completion it is important to briefly note the complications of epigenetic-based therapies as they are an end goal of much epigenetic research. Currently there are therapies used in the clinic that target enzymes responsible for epigenetic alterations, for example DNA methyltransferase inhibitors and histone deacetylase inhibitors (REFs). They are primarily used to treat cancers (REF), but as with many cancer treatments, are highly toxic. These therapies impact the epigenome globally and do not target any specific regions of the genome. This makes them highly undesirable for most diseases and as of yet there are no targeted epigenetic therapies (REF). Methods, such as adapting CRISPR-cas9 enzymes, are being used in laboratories to alter DNA methylation at specific sites (REFs), but it is unclear whether they can be scaled up for clinical use and how long it may take to overcome the various complications.</p>
<p>In summary, there is great potential for EWAS to identify sites in the genome that could be targeted for treatment, but there are several challenges still to overcome. A great importance should be placed on using the data available to inform future designs of EWAS to maximise the potential of these studies.</p>
</div>
</div>
</div>
<div id="genetics-in-ewas" class="section level2" number="1.3">
<h2><span class="header-section-number">1.3</span> Using methods from genetics to help inform future EWAS</h2>
<p>There is a clear corollary to EWAS in genetic epidemiology, the genome-wide association study (GWAS), which also measures markers across the whole genome and assesses whether each of these markers associates with the phenotype of interest. GWAS have existed for longer than EWAS and substantial effort has been put into understanding what information is provided by these studies, what information can be discovered by these studies and how to use these results to inform other research. In this section I briefly describe some examples of these efforts and explain how they might be adapted to help inform future EWAS.</p>
<div id="gwas-catalog" class="section level3" number="1.3.1">
<h3><span class="header-section-number">1.3.1</span> The GWAS Catalog</h3>
<p>The GWAS Catalog is a manually curated database of publicly available GWAS data, developed by the European Bioinformatics Institute (EBI) and made openly available to the public (REFs). It has a broad range of applications for researchers, from replication of GWAS, to identifying overlapping GWAS signals between traits, to pooling the data to try and understand the genetic architecture of complex traits as a whole. Resources like this are invaluable to the genetic epidemiologist community and so developing a corollary database for EWAS may provide equal opportunity for epigenetic epidemiologists. Catalogues such as EWASdb (REF) and the EWAS Atlas (REF) are currently available, but fall short of some key researcher requirements including ease of use and access to full summary statistics. The development of a new database, The EWAS Catalog, is the focus of <strong>Chapter <a href="3-ewas-catalog.html#ewas-catalog">3</a></strong>.</p>
</div>
<div id="heritability" class="section level3" number="1.3.2">
<h3><span class="header-section-number">1.3.2</span> Total variance captured by all sites measured genome-wide</h3>
<p>Before spending lots of time and resources on measuring the relationship between traits, itâs important to have an understanding of whether your exposure of interest covaries with the outcome of interest. Of course, if they are independent then studying the relationship between the two would be pointless. The phenotypic variance of a trait, <span class="math inline">\(\sigma^2_{P}\)</span>, is a function of the environmental variance, <span class="math inline">\(\sigma^2_{E}\)</span>, and genetic variance, <span class="math inline">\(\sigma^2_{G}\)</span>, that contribute to the trait:</p>
<p><span class="math display" id="eq:phenotypic-variance">\[\begin{equation}
    \sigma^2_{P} = \sigma^2_{G} + \sigma^2_{E}
    \tag{1.1}
\end{equation}\]</span></p>
<p><span class="math inline">\(\sigma^2_{G}\)</span> is mostly comprised of the variance due to additive genetic effects (<span class="math inline">\(\sigma^2_{A}\)</span>), and the proportion of phenotypic variance accounted for by the additive genetic variance is the narrow-sense heritability (<span class="math inline">\(h^2\)</span>):</p>
<p><span class="math display" id="eq:narrow-sense-heritability">\[\begin{equation}
    h^2 = \frac{\sigma^2_{A}} {\sigma^2_{P}}
    \tag{1.2}
\end{equation}\]</span></p>
<p>It is possible to estimate <span class="math inline">\(h^2\)</span> by taking advantage of pedigree information in families. This has been done for a plethora of traits to establish that they have some additive genetic component (i.e.Â <span class="math inline">\(\sigma^2_{A}\)</span> &gt; 0 and <span class="math inline">\(h^2\)</span> &gt; 0). This provides adequate evidence that genetic variation influences these traits and thus there are genetic effects to identify.</p>
<p>After the largely unsuccessful attempt of geneticists using candidate gene studies to reliably identify genetic sequence variation that influences traits (REF), a new study design was proposed using arrays to measure hundreds of thousands of genetic variants across the genome in a hypothesis-free approach to identify sites, GWAS. Initially, these studies were conducted in hundreds or thousands of individuals and were identifying very few variants that could be said to reliably have an effect on the trait, and these sites explained an extremely small proportion of the heritability estimates (&lt;1%) (REF). Speculations were made about the reasons why this could be, for example the arrays were only capturing common genetic variation and it was rare genetic variation having the majority of the influence on phenotypes (REF). This provided a need to derive individual aspects of <span class="math inline">\(h^2\)</span> to inform future study design. If indeed common genetic variation contributed little to <span class="math inline">\(h^2\)</span> for the majority of traits, the GWAS approach would need to be re-thought and technologies would need to be made to capture genetic variants not reliably captured by existing arrays (e.g.Â rare variants). Yang et al.Â developed a method to estimate the contribution of all the single nucleotide polymorphisms (SNPs) measured on SNP-arrays to phenotypic variance (SNP-heritability or <span class="math inline">\(h^2_{SNP}\)</span>) (REF). SNP-heritability was subsequently shown not to be inconsequential for complex traits and so warranted continuing use of GWAS and the acquisition of larger samples to conduct these GWAS (REFs).</p>
<p>Lack of associations and lack of phenotypic variance captured by DNA methylation sites has been observed in many EWAS, yet currently there is no corollary test to SNP-heritability to enable increased understanding of why this may be. Of course, there could be a variety of reasons for lack of signal in EWAS, which include what was discussed earlier, e.g.Â measuring the wrong tissue. If there is covariation of DNA methylation with complex traits, it may just be that lots of associations with small effect sizes across the entire genome contribute to this covariation and so EWAS with large sample sizes are required to reliably identify these associations. In <strong>Chapter <a href="5-h2ewas-chapter.html#h2ewas-chapter">5</a></strong>, I apply methods developed to assess SNP-heritability to estimate the proportion of phenotypic variance correlated with DNA methylation across a range of phenotypes.</p>
</div>
<div id="inferring-biology-from-signals" class="section level3" number="1.3.3">
<h3><span class="header-section-number">1.3.3</span> Inferring biology from signals</h3>
<p>As discussed, the complexity of cellular processes makes it difficult to infer what consequences a change in methylation at a specific site may have. Similarly, for the majority of SNPs identified in GWAS, the functional change that relates to an association between genetic variation at that site and the trait of interest is unclear. Traits themselves are the result of a large number of complex biological pathways that are determined by potentially thousands of gene products. It is often assumed that the signal from GWAS highlight genomic regions of importance to the trait and thus as a step to investigate the nature of the signal, sites identified are mapped to nearby genes. These genes can then be mapped to pathways and gene set enrichment analysis performed to assess whether the genes identified are present in any particular pathways more than expected by chance. This method has been adopted by EWAS and is incorporated in various packages that perform EWAS analyses (REFs).</p>
<p>Establishing causality from DNA methylation signal is difficult (<a href="1-introduction.html#dnam-phs">Use of DNA methylation in a population setting</a>, <a href="1-introduction.html#establishing-causality">Establishing causality</a>). Thus, when applying gene set enrichment analyses to identify prominent pathways in EWAS signals, the pathways identified may actually be related to a particular confounder rather than the trait of interest. This is important to remember when interpreting the results of such an analysis, but it doesnât render the results useless. There is a huge body of work that characterises gene action and relationships of this gene action with various traits. By mapping EWAS signals to genes and pathways, a path between the trait (or a confounder) and changes in DNA methylation might become more clear. One example of this comes with EWAS of smoking, that have consistently identified DNA methylation at the <em>AHRR</em> gene. This gene is known to play a role in handling toxic substances found in tobacco smoke. Thus, large changes in DNA methylation related to this gene points towards epigenetic changes at that locus influencing the cellular response to smoking. Of course, some of these links may be tenuous and proving DNA methylation is involved in influencing up or downstream changes of the trait of interest is difficult, but mapping EWAS signal to genes and pathways can add to the pool of information about the underlying biology of the trait and used in conjucture with other studies, may increase understanding of various complex phenotypes.</p>
<p>As DNA methylation changes may be a result of the trait (unlike genetic variation), pathway analysis for EWAS studies could capture biological consequences of the trait. This could allow for EWAS to identify facets of trait biology missed by GWAS studies, however it is unclear as to whether the analogous gene set enrichment design adopted by EWAS is adding to the information discovered by GWAS. In <strong>Chapter <a href="6-ewas-gwas-comp-chapter.html#ewas-gwas-comp-chapter">6</a></strong> I compare overlap of GWAS and EWAS signals in this context and discuss the use of both study designs in elucidating underlying trait biology.</p>
</div>
<div id="establishing-causality" class="section level3" number="1.3.4">
<h3><span class="header-section-number">1.3.4</span> Establishing causality</h3>
<div id="mendelian-randomization" class="section level4" number="1.3.4.1">
<h4><span class="header-section-number">1.3.4.1</span> Mendelian randomization</h4>
<p>As discussed, population-based studies of DNA methylation suffer from the same limitations as any observational epidemiology study, namely confounding and reverse causation. One method that aims to mitigate these limitations is Mendelian randomization (MR) (REFs), which uses genetic variants as proxies for the exposure of interest in an instrumental variable framework (illustrated in <strong>Figure <a href="1-introduction.html#fig:mr-diagram">1.2</a></strong>). Using genetic variants as instruments has the advantage that the direction of effect will always be from instrument to exposure and not <em>vice versa</em>, making interpretation of the studies simpler. Furthermore, unlike environmental phenotypes, that tend to be highly correlated and clustered into groups, genetic variants associated with a trait tend to be unconfounded (REF). In the absence of assortative mating, genetic variants should be distributed randomly across the population, so in effect those grouped by genotype should exhibit differences in exposure, but confounding factors should not differ between genotype groups (REF). Assortative mating has been reliably shown to occur with some traits (REF) and as those traits are social behaviours, such as alcohol consumption, or clear visible traits, such as height, one would assume there is little if any assortative mating with regards to DNA methylation levels so I will ignore assortative mating from this point onwards.</p>
</div>
<div id="availability-of-data-for-mr" class="section level4" number="1.3.4.2">
<h4><span class="header-section-number">1.3.4.2</span> Availability of data for MR</h4>
<p>Another advantage of MR is the data it uses. Thousands of GWAS have been conducted giving researchers ample instruments for a wide variety of traits and many of these instruments are easily accessible through databases such as the GWAS Catalog (REF) and IEU GWAS database (REF). Furthermore, it isnât necessary to use individual-level data to conduct MR studies; summary statistics from GWAS are all that is needed to provide data in a two-sample MR framework (REF). This is especially valuable to conducting MR studies using DNA methylation data, as no cohorts have measured DNA methylation, genotype and a broad range of phenotypes in a large enough sample to provide adequate power to test DNA methylation-phenotype associations in a MR context.</p>
</div>
<div id="assumptions-of-mr" class="section level4" number="1.3.4.3">
<h4><span class="header-section-number">1.3.4.3</span> Assumptions of MR</h4>
<p>There are three main assumptions in MR, these are illustrated in <strong>Figure <a href="1-introduction.html#fig:mr-diagram">1.2</a></strong>. Testing assumption one, the instruments associate with the exposure of interest, is simple, but the other two assumptions canât technically be proven to be true. Horizontal pleiotropy, where genetic variants associate with more variables than just the exposure of interest, can lead to violations in assumptions two and three. Ideally, MR would be performed in the context where the genetic effect on the exposure had been characterised such that the mechanism of action was understood clearly. This would help give evidence against assumptions two and three being broken. Unfortunately, this is rarely possible. However, a plethora of methods have now been developed to test for pleiotropic effects, given the exposure of interest has multiple independent genetic variants reliably associated with it.</p>

<div class="figure" style="text-align: center"><span id="fig:mr-diagram"></span>
<img src="figure/01-introduction/mr-diagram.pdf" alt="Mendelian randomization. Mendelian randomization can be used to test the causal nature of exposure-outcome relationships provided the assumptions are met. Assumption 1. There is an association between the instrument and the exposure. Assumption 2. There are no associations between the instrument and outcome, except through the exposure. Assumption 3. The instrument is not associated with any factors that confound the exposure-outcome relationship."  />
<p class="caption">
Figure 1.2: Mendelian randomization. Mendelian randomization can be used to test the causal nature of exposure-outcome relationships provided the assumptions are met. Assumption 1. There is an association between the instrument and the exposure. Assumption 2. There are no associations between the instrument and outcome, except through the exposure. Assumption 3. The instrument is not associated with any factors that confound the exposure-outcome relationship.
</p>
</div>
</div>
<div id="applying-mr-in-a-dna-methylation-context" class="section level4" number="1.3.4.4">
<h4><span class="header-section-number">1.3.4.4</span> Applying MR in a DNA methylation context</h4>
<p>MR can be applied to studies of DNA methylation by using methylation quantitative trait loci (mQTL), genetic variants associated with changes in DNA methylation levels, as proxies (REF). As mentioned previously using a two-sample MR framework is especially useful to help increase power for these studies (REF). Unfortunately, for each DNA methylation site few independent mQTLs have been identified. This prevents the use of various tests to examine whether the instruments are likely to be pleiotropic. Both cis-mQTLs (mQTLs within 1Mb of the DNA methylation site) and trans-mQTLs (mQTLs over 1Mb away from the DNA methylation site) have been identified in GWAS of DNA methylation variation. As genetic architecture of DNA methylation changes is also not well understood, the mechanism of action for each mQTL can only be speculated at present. Cis-mQTLs are thought to be less likely to be pleiotropic than trans-mQTLs as the mechanism of action seems more likely to be related to the binding of regulatory machinery that may influence DNA methylation levels, for example a genetic variant may decrease the affinity of a transcription factor for that region and so the transcription factor will bind less frequently and/or for a shorter period, this would lead to increased methylation at that site. On the contrary, the mechanism of trans-mQTL action, especially those on separate chromosomes to the DNA methylation site of interest, is more likely to be pleiotropic, for example a trans-mQTL could influence gene expression of a transcription factor that binds many sites and alters their DNA methylation (REFs), this would make the trans-mQTL associate with multiple DNA methylation sites. Therefore, if one limits mQTLs to those in cis, this gives greater confidence that horizontal pleiotropy isnât influencing results.</p>
<p>As mentioned, DNA methylation varies both temporally and across cell types. If the instruments used to represent DNA methylation are viewed as influencing life-time differences in DNA methylation between individuals, then temporal variation can be ignored. However, cell type-specific effects are likely to occur for some mQTLs. One could imagine a scenario in which blood cells require a specific gene (gene A) to be expressed that is completely useless in adipose cells, which could mean all adipose cells have 100% DNA methylation at the promoter region of gene A. Thus, any genetic variants that associate with DNA methylation variation in blood would not also associate with variation in adipose cells.</p>
<p>With all this in mind, itâs important to maintain the idea that making concrete conclusions from DNA methylation is difficult, but triangulating evidence from multiple sources could be key to understanding the role of DNA methylation in underlying trait biology (REFs).</p>
</div>
</div>
</div>
<div id="overview-of-thesis-aims" class="section level2" number="1.4">
<h2><span class="header-section-number">1.4</span> Overview of thesis aims</h2>
<p>DNA methylation has great potential for use in an epidemiological sense and as samples with DNA methylation data continue to grow it is important to understand the limitations of EWAS and how to maximise itâs potential. My thesis aims to address this by exploring what information has been gained from EWAS (<strong>Chapters <a href="3-ewas-catalog.html#ewas-catalog">3</a> and <a href="4-properties-of-ewas.html#properties-of-ewas">4</a></strong>), what information is still to gain from EWAS (<strong>Chapter <a href="5-h2ewas-chapter.html#h2ewas-chapter">5</a></strong>), whether EWAS might add to our biological understanding of complex traits above GWAS (<strong>Chapter <a href="6-ewas-gwas-comp-chapter.html#ewas-gwas-comp-chapter">6</a></strong>) and by applying MR in a particular case, the potential for confounding in EWAS (<strong>Chapter <a href="7-dnam-lung-cancer-mr.html#dnam-lung-cancer-mr">7</a></strong>). See the flowchart in <strong>Figure <a href="1-introduction.html#fig:thesis-flowchart">1.3</a></strong> for a graphical depiction.</p>
<p>In <strong>Chapter <a href="3-ewas-catalog.html#ewas-catalog">3</a></strong> a database of published EWAS is curated and made publicly available, which will be used in later chapters. The aim of <strong>Chapter <a href="4-properties-of-ewas.html#properties-of-ewas">4</a></strong> is to analyse the vast database jointly to allow the discovery of commonalities across methylome-trait associations and provide a platform to explore what is driving these commonalities. Further, the chapter explores the extent to which published results are reliable by assessing replication rate and whether sites measured by unreliable probes are prominent.</p>
<p>After exploring the information already gained from EWAS, <strong>Chapter <a href="5-h2ewas-chapter.html#h2ewas-chapter">5</a></strong> investigates the information still to gain from EWAS. The aim of the chapter is to apply methods developed to assess SNP-heritability to estimate the proportion of complex trait variation that is associated with sites commonly measured in EWAS.</p>
<p><strong>Chapter <a href="6-ewas-gwas-comp-chapter.html#ewas-gwas-comp-chapter">6</a></strong> will then aim to assess whether the discoveries of EWAS may provide extra biological insight for traits of interest on top of those from GWAS. Tests will be applied to assess whether there is more overlap between the sites, genes or pathways identified by some large EWAS (N &gt; 4500) and their corresponding GWAS than expected by chance.</p>
<p>Finally, <strong>Chapter <a href="7-dnam-lung-cancer-mr.html#dnam-lung-cancer-mr">7</a></strong> will apply MR to explore the causal nature of associations between DNA methylation and lung cancer. This application case-study will compare and contrast findings to conventional EWAS estimates to give an example of the potential residual confounding that can be present in EWAS.</p>

<div class="figure" style="text-align: center"><span id="fig:thesis-flowchart"></span>
<img src="figure/01-introduction/thesis-flowchart.pdf" alt="Flowchart showing questions being asked in the thesis and the corresponding chapters attempting to help answer them."  />
<p class="caption">
Figure 1.3: Flowchart showing questions being asked in the thesis and the corresponding chapters attempting to help answer them.
</p>
</div>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="2-methods.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.epub", "EPUB"], ["thesis.docx", "Word"]],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
